docking_node:
  ros__parameters:
    target_x: 1.7 
    target_y: -1.3
    target_theta: -1.6
    approach_distance_threshold: 0.3
    final_approach_distance: 0.2
    docking_distance_threshold: 0.05
    docking_angle_threshold: 0.05

